### PR TITLE
[DOCS] Final review for 2.0.0

### PR DESCRIPTION
**Pull request by frascuchon**
# Description

**Type of change**

- Documentation update

**How Has This Been Tested**

**Checklist**

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: argilla/docs/how_to_guides/workspace.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ A **workspace** is a *space* inside your Argilla instance where authorized users
 
 Argilla provides a default workspace to help you get started in Python and the UI. The name of this workspace varies depending on the server configuration.
 
-| Environment                   | Name |
+| Environment                   | Name     |
 |-------------------------------|----------|
-| Quickstart Docker and HF Space | admin    |
+| Docker and HF Spaces          | admin    |
 | Server image                  | argilla  |
```
Comment (frascuchon) [inline argilla/docs/how_to_guides/workspace.md]: This section does not apply anymore. @sdiazlor did you already change this on https://github.com/argilla-io/argilla/pull/5337?

Comment (sdiazlor) [reply argilla/docs/how_to_guides/workspace.md]: No, not yet. @frascuchon so is there no default workspace either?

Comment (frascuchon) [reply argilla/docs/how_to_guides/workspace.md]: No. For HF spaces the one configured in `.oauth.yaml`. For common docker deployment, the owner must create a workspace first.

Comment (frascuchon) [reply argilla/docs/how_to_guides/workspace.md]: So, maybe replacing this table with the different scenarios depending on the deployment makes sense.

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
 Argilla is a collaboration tool for AI engineers and domain experts who need to build high-quality datasets for their projects.
 
-If you just want to get started, we recommend our [UI demo](https://demo.argilla.io/sign-in?auth=ZGVtbzoxMjM0NTY3OA%3D%3D) or our [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space). Curious, and want to know more? Read our [documentation](https://argilla-io.github.io/argilla/latest/).
+If you just want to get started, we recommend our [UI demo](https://demo.argilla.io/sign-in?auth=ZGVtbzoxMjM0NTY3OA%3D%3D) or our [free Hugging Face Spaces deployment integration](https://huggingface.co/new-space?template=argilla/argilla-template-space). Curious, and want to know more? Read our [documentation](https://docs.argilla.io/latest/).
```
Comment (dvsrepo) [inline README.md]:
```suggestion
If you just want to get started, [deploy Argilla on Hugging Face Spaces](https://docs.v2.argilla.io/latest/getting_started/quickstart/). Curious, and want to know more? Read our [documentation](https://docs.v2.argilla.io/latest/).
```

**Pull request merged**